vscode-dotnet-runtime-library/src/IDotnetListVersionsContext.ts

```diff
@@ -0,0 +1,50 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+export interface IDotnetListVersionsContext {
+    /**
+     * @remarks
+     * Holds the parameters necessary to acquire the available SDK or Runtimes.
+     * By available this means only those at the newest minor version for each major version.
+     *
+     * @property listRuntimes - If this is not true (false or undefined), list SDK versions available. Elsewise, list runtimes available.
```

Comment:
That's a good point. The reason is that the Global SDK extension will need to be able to install Runtimes and list the runtimes on the machine. And we're still discussing how to brand the changes (either a new extension, rebrand the runtime one, or rebrand the SDK one.) I talked to Chet and he suggested we keep both in the SDK but not in the runtime extension.

Reply:
I don't actually think there's a good way to 'hide' the boolean from the runtime extension if one of the extensions (the sdk one) needs both. We could error if you try to acquire SDKs from the runtime extension, but I don't really think that's super helpful, because you'd still see the ability to acquire SDKs from the runtime extension, it just would throw on you. For now I'm going to leave this as-is but feel free to chime in Sarah or @baronfel if you strongly disagree.

Reply:
Ok. We just had a meeting we're going to remove this functionality from the runtime.